src/slides/redux-intro/reducer-example.js

```diff
@@ -9,7 +9,7 @@ const visibilityFilter = (state = "SHOW_ALL", action) => {
 const todos = (state = [], action) => {
   switch (action.type) {
     case "ADD_TODO":
-      return state.concat([{ text: action.text, completed: false }]);
+      return state.todos.concat([{ text: action.text, completed: false }]);
```

Comment:
Should this be `todos`?

Reply:
i think its just state because we're using `combineReducers`

Reply:
Good point. I will fix it.